times.go

```diff
@@ -25,28 +25,38 @@ func Time(then time.Time) string {
 	return RelTime(then, time.Now(), "ago", "from now")
 }
 
-var magnitudes = []struct {
+type Magnitude struct {
 	d      int64
 	format string
 	divby  int64
-}{
-	{1, "now", 1},
-	{2, "1 second %s", 1},
-	{Minute, "%d seconds %s", 1},
-	{2 * Minute, "1 minute %s", 1},
-	{Hour, "%d minutes %s", Minute},
-	{2 * Hour, "1 hour %s", 1},
-	{Day, "%d hours %s", Hour},
-	{2 * Day, "1 day %s", 1},
-	{Week, "%d days %s", Day},
-	{2 * Week, "1 week %s", 1},
-	{Month, "%d weeks %s", Week},
-	{2 * Month, "1 month %s", 1},
-	{Year, "%d months %s", Month},
-	{18 * Month, "1 year %s", 1},
-	{2 * Year, "2 years %s", 1},
-	{LongTime, "%d years %s", Year},
-	{math.MaxInt64, "a long while %s", 1},
+}
+
+func NewMagnitude(d int64, format string, divby int64) Magnitude {
```

Comment:
`NewMagnitude` is an exported identifier and should be documented according to [Go style](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments).

Otherwise, `golint` would complain:

```
exported type Magnitude should have comment or be unexported
exported function NewMagnitude should have comment or be unexported
```